_layouts/default.html

```diff
@@ -1,16 +1,13 @@
 <!DOCTYPE html>
```

Comment:
To use the default layout, other page just needs to specify the front matter:
---
layout: default
sections:
- name: section1.html
- name: section2.html
---